packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_table_row.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:flutter/material.dart';
+
+import 'logging_controller_v2.dart';
+
+class LoggingTableRow extends StatefulWidget {
+  const LoggingTableRow({
+    super.key,
+    required this.index,
+    required this.data,
+    required this.isSelected,
+  });
+
+  final int index;
+  final LogDataV2 data;
+  final bool isSelected;
+
+  @override
+  State<LoggingTableRow> createState() => _LoggingTableRowState();
+
+  static TextSpan _detailsSpan(String text) {
+    return TextSpan(
+      text: text,
+      style: _detailsStyle,
+    );
+  }
+
+  static TextSpan _metadataSpan(String text) {
+    return TextSpan(
+      text: text,
+      style: _metaDataStyle,
+    );
+  }
+
+  static const _detailsStyle = TextStyle();
+
+  static const _metaDataStyle = TextStyle();
```

Comment:
why do we need these empty styles?

Reply:
I put these here so I could hand bake the text styles.
The model and the table both need access to the text styles so that we can calculate the proper height.

Getting the BuildContext is kind of weird though outside of the widget tree.

Do you know of any better ways to just get the default textStyles without having to hand bake them?

Reply:
Looking at other areas in the code, we may just have to define it directly here. That being said, I think we can remove these 4 static members related to text style and text span, and just have the calculation methods below write:

`TextSpan(text: text, style: TextStyle(fontSize: defaultFontSize))`